src/main/resources/migrations/mariadb/V1_13_4__Better_Ticket_Carry_Linking.sql

```
@@ -0,0 +1,19 @@
+create schema if not exists `dungeon-hub`;
+use `dungeon-hub`;
+
+alter table ticket_panel
+    add column related_carry_tier BIGINT REFERENCES carry_tier (id);
+
+alter table ticket_panel
+    add column related_carry_difficulty BIGINT REFERENCES carry_difficulty (id);
+
+update ticket_panel
+    set ticket_panel.related_carry_tier = (
+        select carry_tier.id from carry_tier where related_ticket_panel = ticket_panel.id
```

SQLFluff (4.2.1)

[error] The 'where' keyword should always start a new line.

(LT14)